### ISSUE TITLE
Import content from current site

### ISSUE THREAD

**Issue by DeniseColbert** [FreeUKGen/FreeUKGenealogyWP#147]

**Comment by johnshaw1000** [comment]
Is there anything left to do on this?

**Comment by DeniseColbert** [comment]
Yes, lots! The News section has been imported but the static pages are a different matter. @kwheeler08 is bringing over some of the discrete sections currently, but other pages need designing/signing off before their turn comes.
